lib/config.js

```diff
@@ -507,47 +484,36 @@ const loadComponents = (config, type) => {
           return b.sortingLevel - a.sortingLevel;
         }
       })
-      .reduce(((flat, component) => flat.concat(component.files)), []);
+      .reduce((flat, component) => flat.concat(component.files), []);
     return {components, order};
   }, error => {
-    const errStr = error.toString();
     if (error.code === 'NO_BOWER_JSON') {
       logger.error('You probably need to execute `bower install` here.', error);
-    } else if (!enoentRe.test(errStr)) {
+    } else if (!/ENOENT/.test(error)) {
       logger.error(error);
     }
 
-      // Returning default values
+    // Returning default values
     return {components: [], aliases: [], order: []};
   });
 };
 
 const loadNpm = (config) => {
   return new Promise((resolve, reject) => {
+    const paths = config.paths;
+    const rootPath = sysPath.resolve(paths.root);
```

Comment:
If it does, promise will be rejected. No need to do extra `try/catch` and call `reject`.